piqe_ocp_lib/api/resources/ocp_resource_quotas.py

```diff
@@ -0,0 +1,926 @@
+from piqe_ocp_lib.api.resources.ocp_base import OcpBase
+from piqe_ocp_lib.api.ocp_exceptions import ConfigError
+from kubernetes.client.rest import ApiException
+from kubernetes.client import V1ResourceQuota, V1ResourceQuotaSpec, V1ResourceQuotaStatus, V1ObjectMeta, \
+    V1ScopeSelector, V1ScopedResourceSelectorRequirement
+from openshift.dynamic.resource import ResourceList, ResourceInstance
+from typing import List, Union, Dict, Type, Any, Optional
+from enum import Enum, auto
+import logging
+from piqe_ocp_lib import __loggername__
+
+logger = logging.getLogger(__loggername__)
+
+
+class OcpResourceQuota(OcpBase):
+    """
+    OcpResourceQuota Class extends OcpBase and encapsulates all methods
+    related to openshift resource quotas.
+    :param kube_config_file: A kubernetes config file.
+    :return: None
+    """
+    def __init__(self, kube_config_file=None):
+        super(OcpResourceQuota, self).__init__(kube_config_file=kube_config_file)
+        self.api_version = 'v1'
+        self.kind = 'ResourceQuota'
+        self.ocp_resource_quota = self.dyn_client.resources.get(api_version=self.api_version, kind=self.kind)
+
+    def get_resource_quotas(self, namespace: str) -> ResourceList:
```

Comment:
Since this method can return either a ResourceList or None, should then this be -> Union[None, ResourceList]?